<!-- This is a file to control the triskar robot using a PS3-like Joystick. What is different here
     is the fact that I can increase velocity using the joystick not the keyboard.-->

<launch>

	<node pkg="joy" type="joy_node" name="joystick">
        <param name="autorepeat_rate" value="60" />
	</node>

	<node pkg="robogame_joystick" type="robogame_joystick_node" name="robogame_joystick" output="screen">
		<param name="linear_scale" value="1" />
		<param name="angular_scale" value="2.57" />
		<param name="axis_deadman" value="5"/>
		<param name="linear_axis" value="1"/>
		<param name="angular_axis" value="2"/>

	</node>
</launch>
